runtime/src/weights/pallet_grandpa.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2024, Horizen Labs, Inc.
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Default weights for the Grandpa Pallet
+//! This file was not auto-generated.
+
+use core::marker::PhantomData;
```

Comment:
Can you add some comment of how you had computed them?

Reply:
Sure thing, done